tests/unit/test_deleteuser.py

```diff
@@ -0,0 +1,114 @@
+
+import unittest
+from uuid import UUID
+import grpc
+from unittest.mock import patch
+import logging
+
+import os
+import sys
+
+from datetime import date
+
+from sqlalchemy.sql.functions import user
+currentdir = os.path.dirname(os.path.realpath(__file__))
+parentdir = os.path.dirname(currentdir)
+grandparentdir = os.path.dirname(parentdir)
+sys.path.append(grandparentdir)
+
+from server import UserServicer, user_pb2, ValidationFailed
+from model.user_model import User
+
+
+# Disable logging details. You can highlight code to include logging details back.
+logging.disable(logging.CRITICAL)
+
+
+class TestDeleteUser(unittest.TestCase):
+
+    def test_valid_id(self):
+        class UserControllerMock:
+            def add_user(self, user):
```

Comment:
you don't need to define this to test `DeleteUser`. Also the name of the test is `test_valid_id` but that's not what ytou are testing, you are testing `DeleteUser`